### PR TITLE
Correct usage of bitbucket typed repository

### PR DESCRIPTION
**Pull request by noha**
Iceberg repository should return MCBitbucketRepository instead of MCGithubRepository

### CODE REVIEW

File: Iceberg.package/MCBitbucketRepository.extension/instance/getOrCreateIcebergRepository.st [hunk 1]
```diff
@@ -0,0 +1,15 @@
+*Iceberg-Adapters
+getOrCreateIcebergRepository
```
Comment (estebanlm) [inline Iceberg.package/MCBitbucketRepository.extension/instance/getOrCreateIcebergRepository.st]: shouldn't this be moved to its parent?

Comment (noha) [reply Iceberg.package/MCBitbucketRepository.extension/instance/getOrCreateIcebergRepository.st]: I was asking that myself but wasn't sure

Comment (estebanlm) [reply Iceberg.package/MCBitbucketRepository.extension/instance/getOrCreateIcebergRepository.st]: so?

Comment (noha) [reply Iceberg.package/MCBitbucketRepository.extension/instance/getOrCreateIcebergRepository.st]: I can move it up

**Comment by estebanlm** [comment]
btw, this will close #481 ?

File: Iceberg.package/MCBitbucketRepository.extension/instance/getOrCreateIcebergRepository.st (hunk 1 above)
Comment (noha) [reply Iceberg.package/MCBitbucketRepository.extension/instance/getOrCreateIcebergRepository.st]: done

**Comment by noha** [comment]
I'm not sure what #481 is about exactly but it sounds that it is the same case. It is a two-fold fix. The first is in libgit accepting paths with leading slashes and thus enable multi-level dirs. The second fix is that a bitbucket instance is used instead of github. This way you can use bitbucket://user/project in metacello

**Comment by estebanlm** [comment]
yes it is

**Pull request merged**